modular_skyrat/modules/loadouts/loadout_ui/loadout_manager.dm

```diff
@@ -169,8 +165,9 @@
 		allowed_configs += "[initial(colored_item.greyscale_config_inhand_right)]"
 
 	var/slot_starting_colors = initial(colored_item.greyscale_colors)
-	if(INFO_GREYSCALE in owner.prefs.loadout_list[colored_item])
-		slot_starting_colors = owner.prefs.loadout_list[colored_item][INFO_GREYSCALE]
+	if(colored_item in owner.prefs.loadout_list)
+		if(INFO_GREYSCALE in owner.prefs.loadout_list[colored_item])
```

Comment:
You could combine this into one single `if()` statement

Reply:
Yeah this.

Reply:
If I had to guess, it would be because the `in` is trying to evaluate everything that comes after it potentially.